src/styles/antd-overrides/input.scss

```diff
@@ -242,12 +242,15 @@ textarea.ant-input::placeholder {
 
 
 .ant-upload {
-  color: var(--text-primary)
+  color: var(--text-primary);
+  min-height: 101px;
```

Comment:
**question**: is there a solution where this isn't needed?

Generally want to avoid:
- hyper-specific sizes
- pixels
- messing with antd too much

Reply:
Yeah, antd hardcodes the height itself, and because I wanted to increase the image preview size a bit more, I needed to increase the size of upload dropbox container